database/migrations/2024_04_23_103334_add_user_info_to_orders.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('orders', function (Blueprint $table) {
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('order_status_id')->default(4);
```

Comment:
Deze kan dus weg uit de database, als je met dev heb gemerged dan is daar al een migratie die volgensmij ook al een user_id + status toevoegt aan de tabel